### PR TITLE
CO2 Internation data compilation.

### PR DESCRIPTION
**Pull request by kylemasc917**
For this assignment all pull requests going forward should contain the following (please delete the placeholder text and add your own description)

# Title
  - CO2 International data

## Linting
  -This does pylint 10/10

## Contents
Txt files containing data from an observatory on each continent from 2013 to 2023
.json compiled file of all data
python module that converts the txt data into a compiled .json
NOTES file that contains the citation and license for data

## Labels and Milestones
  - ALL pull requests must have an attached label and Milestone before a reviewer can be assigned.

## Reviewer
  - avgagliardo

Any further questions on formatting pull requests can be asked in the issues tab or on discord. 
This document can be changed at any time.

### CODE REVIEW

**Comment by laserlab** [comment]
@zbpetersbuf Do you want to review this, then you can see if it fits your function

**Comment by zbpetersbuf** [comment]
ya ill review it

**Comment by laserlab** [comment]
@kylemasc917 Let's do the same approach with the NOTES file as in PR #36

File: CO2_International_Data/Balkan_Data.txt
Comment (zbpetersbuf) [inline CO2_International_Data/Balkan_Data.txt:1]: funny, approved

File: CO2_International_Data/international_data_converter.py
Comment (zbpetersbuf) [inline CO2_International_Data/international_data_converter.py:1]: everything looks good for me

**Review by zbpetersbuf — APPROVED** [review]
I approve this data

**Comment by SchrodingersStruggle** [comment]
> there are some issues with pylons such as lines being too long but that doesn't effect the efectivness of these files, also unused pandas import that could be deleted but since it functions with our without it being there there's no deire need to delete it so good with me

@kylemasc917 
If you'd wish to make any changes messaged here just create a quick pull request for smaller additions

**Pull request merged**